eap6-support/hawkular-wildfly-agent-feature-pack-eap6/src/main/resources/subsystem-templates/hawkular-wildfly-agent.xml

```diff
@@ -622,11 +623,11 @@
         <operation-dmr name="Shutdown" internal-name="shutdown" > <!-- shutdown and restart -->
```

Comment:
Shouldn't be those `{start,restart,stop}-servers`, `restart`, `reload`, `stop` operations marked also with the modifies=true? Does it make sense to stop/start a server in a container instead of stopping, starting the whole container or pod?